### PR TITLE
feat : added profile  page

### PR DESCRIPTION
**Pull request by Priya-Patel31**

### CODE REVIEW

**Review by t007rushi — APPROVED** [review]
Just look for good naming convention

File: profile_page/profile_page.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    color: var(--light-color);
+  }
+  .profile-content-container {
+    display: grid;
+    grid-template-columns: 15.62rem auto;
+  }
+  .user-profile-container {
+    grid-column-start: 1;
+    grid-column-end: 2;
+    margin: 1rem 0.5rem;
```
Comment (Ankur9669) [inline profile_page/profile_page.css:40]: Suggestion: You can use grid-column

File: profile_page/profile_page.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    justify-content: center;
+  }
+  .component-image-rounded img {
+    width: 12rem;
+    height: 12rem;
+  }
+  .user-details-container {
+    grid-column-start: 2;
+    grid-column-end: 3;
+    display: flex;
```
Comment (Ankur9669) [inline profile_page/profile_page.css:51]: Suggestion: You can use grid-column

File: profile_page/profile_page.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      display: grid;
+      grid-template-columns: 1fr;
+      grid-template-rows: 12.5rem auto;
+    }
+    .user-profile-container {
+      grid-row-start: 1;
+      grid-row-end: 2;
+      grid-column-start: 1;
+      grid-column-end: 2;
+    }
```
Comment (Ankur9669) [inline profile_page/profile_page.css:67]: Suggestion: You can use grid-column, you can use grid-row

**Review by Ankur9669 — APPROVED** [review]
Code looks good to merge. There are some suggestions given by me that is optional,  Great work!!!

File: profile_page/profile_page.css (hunk 1 above)
Comment (Priya-Patel31) [reply profile_page/profile_page.css:40]: but I don't understand that shorthand and I tend to make mistake in that case . That is why used this .

File: profile_page/profile_page.css (hunk 2 above)
Comment (Priya-Patel31) [reply profile_page/profile_page.css:51]: same as above

File: profile_page/profile_page.css (hunk 3 above)
Comment (Priya-Patel31) [reply profile_page/profile_page.css:67]: same as above

**Pull request merged**